core/runtime.rs

```diff
@@ -320,6 +322,11 @@ impl JsRuntime {
     self.v8_isolate.as_mut().unwrap()
   }
 
+  pub fn handle_scope(&mut self) -> v8::HandleScope {
+    let context = self.global_context();
+    v8::HandleScope::with_context(self.v8_isolate(), context)
+  }
```

Comment:
I think this should be an internal helper method (`pub(crate)`) rather than part of the public interface. Usually callers should not be interacting with raw V8 HandleScopes.

Reply:
Agreed, and yet benchmarks are interacting - I think it's better than having to use the code from this function directly